draft-ietf-add-ddr.md

```diff
@@ -225,12 +225,12 @@ Details of such policy are out of scope of this document. Clients SHOULD NOT
 automatically use a Designated Resolver without some sort of validation,
 such as the two methods defined in this document or a future mechanism.
 
-## Authenticated Discovery {#authenticated}
+## Validated Discovery {#validated}
 
-Authenticated Discovery is a mechanism that allows automatic use of a
+Validated Discovery is a mechanism that allows automatic use of a
 Designated Resolver that supports DNS encryption that performs a TLS handshake.
 
-In order to be considered an authenticated Designated Resolver, the
+In order to be considered a validated Designated Resolver, the
 TLS certificate presented by the Designated Resolver MUST contain the IP address
 of the designating Unencrypted Resolver within the SubjectAlternativeName certificate
 field. If the certificate can be validated, the client SHOULD use the discovered
```

Comment:
The use of validated here might be worth looking at.  I don't know if you have other uses.  RFC 6125 uses "verified", so maybe that is better.

Reply:
Yeah, verified can work too. Let's try that out.

Reply:
Okay, changed to verified—I had been debating between validated and verified to begin with.